pom.xml

```diff
@@ -136,7 +136,7 @@
     <derby.version>10.12.1.1</derby.version>
     <parquet.version>1.8.2</parquet.version>
     <hive.parquet.version>1.6.0</hive.parquet.version>
-    <jetty.version>9.2.16.v20160414</jetty.version>
+    <jetty.version>9.3.11.v20160721</jetty.version>
```

Comment:
Ah, thanks, there was some confusion about this. I have fixed the JIRA title and the PR title to match up with the actual code - which uses the exact same version as Apache Hadoop (9.3.11.v20160721).